mirdata/datasets/da_tacos.py

```diff
@@ -0,0 +1,568 @@
+    ),
+    "coveranalysis_hpcp": download_utils.RemoteFileMetadata(
+        filename="da-tacos_coveranalysis_subset_hpcp.zip",
+        url="https://zenodo.org/record/3520368/files/da-tacos_coveranalysis_subset_hpcp.zip?download=1",
+        checksum="961784fc2419214adf05504e9fc56cc2",
+        destination_dir=".",
+    ),
+    "coveranalysis_key": download_utils.RemoteFileMetadata(
+        filename="da-tacos_coveranalysis_subset_key.zip",
+        url="https://zenodo.org/record/3520368/files/da-tacos_coveranalysis_subset_key.zip?download=1",
+        checksum="6e72db855bad5805a67382bd318eee9c",
+        destination_dir=".",
+    ),
+    "coveranalysis_madmom": download_utils.RemoteFileMetadata(
+        filename="da-tacos_coveranalysis_subset_madmom.zip",
+        url="https://zenodo.org/record/3520368/files/da-tacos_coveranalysis_subset_madmom.zip?download=1",
+        checksum="42482eedfe9d9a8be9db3611b9d343b4",
+        destination_dir=".",
+    ),
+    "coveranalysis_mfcc": download_utils.RemoteFileMetadata(
+        filename="da-tacos_coveranalysis_subset_mfcc.zip",
+        url="https://zenodo.org/record/3520368/files/da-tacos_coveranalysis_subset_mfcc.zip?download=1",
+        checksum="11371910cad7012daaa81a5fe9dfa1c0",
+        destination_dir=".",
+    ),
+}
+
+
+class Track(core.Track):
+    """da_tacos track class
+
+    Args:
+        track_id (str): track id of the track
+
+    Attributes:
+        cens_path (str): cens annotation path
+        crema_path (str): crema annotation path
+        hpcp_path (str): hpcp annotation path
+        key_path (str): key annotation path
+        madmom_path (str): madmom annotation path
+        mfcc_path (str): mfcc annotation path
+        tags_path (str): tags annotation path
+        track_id (str): track id
+
+    """
+
+    def __init__(self, track_id, data_home):
+        if track_id not in DATA.index["tracks"]:
+            raise ValueError(
+                "{} is not a valid track ID in da-tacos dataset".format(track_id)
+            )
+
+        self.track_id = track_id
+        self._data_home = data_home
+        self._track_paths = DATA.index["tracks"][track_id]
+        self.cens_path = os.path.join(self._data_home, self._track_paths["cens"][0])
+        self.crema_path = os.path.join(self._data_home, self._track_paths["crema"][0])
+        self.hpcp_path = os.path.join(self._data_home, self._track_paths["hpcp"][0])
+        self.key_path = os.path.join(self._data_home, self._track_paths["key"][0])
+        self.madmom_path = os.path.join(self._data_home, self._track_paths["madmom"][0])
+        self.mfcc_path = os.path.join(self._data_home, self._track_paths["mfcc"][0])
+        self.tags_path = core.none_path_join(
+            [self._data_home, self._track_paths["tags"][0]]
+        )
+        # metadata
+        metadata = DATA.metadata(data_home)
+        self._track_metadata = (
+            metadata[track_id] if track_id and metadata is not None in metadata else {}
```

Comment:
@PRamoneda what is this line meant to do?

Reply:
done